extensions/wikia/AdEngine/js/pageFairDetection.js

```diff
@@ -50,8 +50,19 @@ define('ext.wikia.adEngine.pageFairDetection', [
 		dispatchDetectionEvent(eventName);
 	}
 
+	function canBeInitialized(injectedContext) {
+		return injectedContext !== undefined &&
+			   injectedContext.hasOwnProperty('opts') &&
```

Comment:
why not  `injectedContext && injectedContext.opts && injectedContext.opts.pageFairDetection`?